CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+Contributing to the Node.js Hardware Working Group
+==================================================
+
+## Code of Conduct
+
+### Purpose
+
+The core of the NodeBots community is the people in it. Our primary goal is ensure this community is as inclusive to the largest number of people, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, age, socioeconomic status, religion (or lack thereof), or other marginalized aspect of attendees.
```

Comment:
Should the NodeBots community reference here be changed to "Node.js Hardware Working Group"?